src/main/java/org/fundacionjala/coding/juan/eanvalidator/EanValidator.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.juan.eanvalidator;
+
+/**
+ * @author Juan Pablo
+ */
+public final class EanValidator {
+    /**
+     * constructor.
+     */
+    private EanValidator() {
+
+    }
+
+    /**
+     * @param eanCode is the code provided as input.
+     * @return the validation.
+     */
+
+    public static boolean validating(final String eanCode) {
+        int sum = 0;
+        int checksum = 0;
+        final int finalNumber = 12;
+        final int multiply = 3;
+        final int finalFactor = 10;
+        final int aux = 48;
```

Comment:
@AT-03/at-03 
why are you using 48 for ascii code?
try to avoid that in your solution.